### PR TITLE
Setup repo

### PR DESCRIPTION
**Pull request by nechaido**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@metarhia/metagram",
+  "version": "1.0.0",
+  "description": "ERD generator for metaschema",
+  "main": "metagram.js",
```
Comment (tshemsedinov) [inline package.json:5]: Add keys:
- files
- types

**Review by tshemsedinov — COMMENTED** [review]
Add editor config like this https://github.com/metarhia/metasql/blob/master/.editorconfig

File: package.json (hunk 1 above)
Comment (nechaido) [reply package.json:5]: I have added `files` but see no point in `types`.
1. It is not a library, it's a cli tool.
2. There are no types. If package grows bigger we can go on and add typings, but now I do not see a reason.

Comment (tshemsedinov) [reply package.json:5]: If we will use it just as a cli tool it's ok, but in future I'd like to generate visualization in runtime from UI

**Review by tshemsedinov — APPROVED** [review]

**Pull request merged**